iso.bbx

```diff
@@ -113,63 +133,92 @@
   {\item}%
 %
 
-%field formats
+% FIELD FORMATS
 
+% Format title and subtitle as is
 \DeclareFieldFormat*{title}{#1}
 \DeclareFieldFormat*{subtitle}{#1}
+
+% Format volume field with short localisation string
 \DeclareFieldFormat{volume}{\bibsstring{volume}\addnbspace#1}% volume of a book
+% Differentiate volume in journal
 \DeclareFieldFormat[article,periodical]{volume}{%
   \iftoggle{bbx:shortnum}
     {\mkbibbold{#1}}
     {\bibsstring{jourvol}\addnbspace#1}% volume of a journal
 }
+
+% Edition field could be:
+% a) numeral: use with localisation string
+% b) literal: capitalize given string
 \DeclareFieldFormat*{edition}{%
   \ifnumeral{#1}%
     {\mkbibordedition{#1}\addnbspace\bibsstring{edition}}%
     {\MakeCapital{#1}}%
 }%
+
+% Format pagination for page range
 \DeclareFieldFormat*{pages}{\mkmlpageprefix[bookpagination]{#1}}
+% Differentiate page range in journal
 \DeclareFieldFormat[article,periodical]{pages}{%
   \iftoggle{bbx:shortnum}
     {#1}
     {\mainsstring{pages}\addspace\printtext{#1}}%
 }
+
+% Format pagination for total number of pages
 \DeclareFieldFormat*{pagetotal}{\mkbibbrackets{\mkmlpagetotal[bookpagination]{#1}}}
+
+% Format number field with short localisation string
 \DeclareFieldFormat*{number}{\bibsstring{number}\addspace\printtext{#1}}
+% Differentiate number in journal
 \DeclareFieldFormat[article,periodical]{number}{%
   \iftoggle{bbx:shortnum}
     {\mkbibparens{#1}}
     {\bibsstring{number}\addspace\printtext{#1}}%
 }
+% Differentiate number for patent: format as is
 \DeclareFieldFormat[patent]{number}{#1}
+
+% a) url AND no urldate: use localisation string 'available ALSO from',
+%    meaning: the work is not primarily published electronically
+% b) url AND urldate: use localisation string 'available from'
 \DeclareFieldFormat*{url}{%
   \iffieldundef{urlyear}
```

Comment:
It would be possible to shorten this to
```latex
\DeclareFieldFormat*{url}{%
  \iffieldundef{urlyear}
    {\mainlstring{urlalso}}
    {\mainlstring{urlfrom}}%
  \addcolon\space\url{#1}%
}
```